README.md

```diff
@@ -43,9 +43,9 @@ Once complete, the Datadog Agent is started automatically when each dyno starts.
 The Datadog Agent provides a listening port on `8125` for statsd/dogstatsd metrics and events. Traces are collected on port `8126`.
 
 ### Order of buildpacks
-As explained in [Heroku's documentation][4], the last buildpack in the list is used to determine the process type for the application.
+As explained in the [Heroku][4] documentation, the last buildpack in the list is used to determine the process type for the application.
```

Comment:
I think the word "documentation" should be part of the link text here because the link takes you to Heroku documentation, not to Heroku. I understand the spirit of this lint rule is to keep the link text more descriptive than just "documentation", but you should be able to say "the X documentation" for the link text.

Reply:
Made the link more specific since it links to an article instead of the main page of the documentation.